### ISSUE TITLE
Add some documentation

### ISSUE THREAD

**Issue by zonca** [CMB-S4/s4sim#9]
@keskitalo @smsimon 

I think we need some documentation

even just adding something to https://github.com/CMB-S4/s4sim/blob/master/README.md

In particular about the hardware model:

* what is the difference between `config` `config_1shooter` and 3 shooter
* how do I get the baseline design
* should we rename `get_example` to `get_baseline_design`?

**Comment by smsimon** [comment]
Agreed, this should be documented. We should think about the right way to do it since we really only want to use the baseline. I think it would be fine to change get_example to get_baseline_design. The main code only calls config.py, which is the baseline. 

There were several versions of configurations that I made before we got confirmation from the SAT group on what the baseline was, but we didn't want to delete them in case they'd be useful for the future. 1 shooter is all individual SATs (i.e. all have independent scanning), 3 shooter is a SAT with three tubes (no independent scanning) but the tubes are not focused on the same area of the sky (which could be useful in the future if there are slight offsets on the area of the sky that the tubes see in future iterations).
